scripts/apps/dashboard/user-activity/components/UserActivityWidget.tsx

```diff
@@ -126,21 +102,34 @@ const GET_GROUPS = (userId, services: any): Array<IGroup> => {
         {
             id: 'marked',
             label: gettext('Marked for this user'),
-            dataSource: {
-                repo: 'archive',
-                query: {...getQueryMarkedForUser(userId)},
+            precondition() {
+                return Object.values(extensions)
+                    .some(({extension}) =>
```

Comment:
You should be getting the extension by ID and operating on that single extension. It's not valid for other extensions to expose mark-for-user related methods, but by reading your code one would think that it is, since you are reading the method from all the extensions.